test/sequential/test-inspector.js

```diff
@@ -8,7 +8,12 @@ const assert = require('assert');
 const { NodeInstance } = require('../common/inspector-helper.js');
 
 function checkListResponse(response) {
-  assert.strictEqual(1, response.length);
+  const expectedResponseLength = 1;
+  assert.strictEqual(
+    response.length,
+    expectedResponseLength,
+    `Expected list response length to be ${expectedResponseLength}.`
```

Comment:
Nit: this removes important information because we do not know what the actual value actually is.

Reply:
I see that now. :) 
Is it worth pushing a small fix for this? (It's easy enough for me to do, but I'm not sure if that will mess up your review process...)

Reply:
@JeffMarvin It won't mess up the process, [here are instructions](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#step-9-discuss-and-update) on how to update the PR

We'll squash the commits before pushing it to master